source/use-cases/Untitled-1

```diff
@@ -0,0 +1 @@
+mgeneratejs '{"name": "$name", "address": "$address", "phone": { "$integer": {"min": 3471111111, "max":8009999999 }, "ssn":{"$integer": {"min": 111111111, "max": 999999999}}, "ssnLast4": "0000", "bloodType": { "$choose": { "from": ["ab-", "b-","ab+","a-"], weight: [1,1,1,1]}} } ,"insurance": "policyNumber": { "$integer": {"min": 11111, "max":99999 } , "provider": {"$choose": { "from": ["MedSilver Insure", "HealthAid Pro", "Caresimple Co", "HealthConnect Insure"], "weights": [1,1,1,1] } } }, "medicalRecords": {"$array": {"of": {"weight": "{{chance.natural({min: 160, max: 180})}}", "heartRate": "{{chance.natural({min: 70, max: 100})}}", "bloodPressure": { "$choose": { "from": ["120/80", "115/75", "120/70"], weight: [1,1,1]}} }, "number": "{{chance.natural({ min: 1, max: 10})}}" }} }'  -n 1000
```

Comment:
Did you mean to commit this file? I was not able to find it referenced in the article.

Reply:
Sorry about that, I've removed it now